group_vars/pulsarservers.yml

```diff
@@ -0,0 +1,53 @@
+
+galaxy_server_hostname: "galaxy.atgm.avans.nl" # Important!!!
+# Put your Galaxy server's fully qualified domain name (FQDN) (or the FQDN of the RabbitMQ server) above.
+
+pulsar_root: /mnt/pulsar
+
+pulsar_pip_install: true
+pulsar_pycurl_ssl_library: openssl
+pulsar_systemd: true
+pulsar_systemd_runner: webless
+
+pulsar_create_user: true
+pulsar_user: {name: pulsar, shell: /bin/bash}
+
+pulsar_optional_dependencies:
+  - pyOpenSSL
+  # For remote transfers initiated on the Pulsar end rather than the Galaxy end
+  - pycurl
+  # drmaa required if connecting to an external DRM using it.
+  - drmaa
+  # kombu needed if using a message queue
+  - kombu
+  # amqp 5.0.3 changes behaviour in an unexpected way, pin for now.
+  - 'amqp==5.0.2'
+  # psutil and pylockfile are optional dependencies but can make Pulsar
+  # more robust in small ways.
+  - psutil
+
+pulsar_yaml_config:
+  staging_directory: "{{ pulsar_staging_dir }}"
+  persistence_directory: "{{ pulsar_persistence_dir }}"
+  tool_dependency_dir: "{{ pulsar_dependencies_dir }}"
+  # The following are the settings for the pulsar server to contact the message queue with related timeouts etc.
+  message_queue_url: "pyamqp://galaxy_au:{{ vault_rabbitmq_password_vhost }}@{{ galaxy_server_hostname }}:5671//pulsar/galaxy_au?ssl=1"

```

Comment:
`galaxy_au` so I think this needs to be templated out, since you've named the queues after the machines, it should be matchingly templated per machine. In the host_vars you can add something like `machine_shortname: midgard` and then use that variable here?

Reply:
Good catch, also updated the vault password since i only use pulsar_password